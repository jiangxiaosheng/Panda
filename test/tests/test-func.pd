func foo(a: int): void {
	for var i = 0; i < a; i + 1 {
		print("" + a)
		print("")
	}

}

func main() {
    foo(10)
}