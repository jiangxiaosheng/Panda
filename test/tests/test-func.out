

Sementically checked program: 


void foo(a) {
for (var i: int(int : 0)
; (bool : (int : i) < (int : a)); (int : (int : i) + (int : 1))) {
(int : print((string : (string : ) + (string : a))))
(int : print((string : )))
}
}

void main() {
(void : foo((int : 10)))
}

